config/environments/production.rb

```diff
@@ -80,11 +79,5 @@
   config.active_support.deprecation = :notify
 
   # Use default logging formatter so that PID and timestamp are not suppressed.
-  config.log_formatter = ::Logger::Formatter.new
-
-  $real_stdout = $stdout.clone
-  $stdout.reopen($stderr)
-  config.logstasher.enabled = true
-  config.logstasher.logger = Logger.new($real_stdout)
-  config.logstasher.suppress_app_log = true
+  # config.log_formatter = ::Logger::Formatter.new
```

Comment:
Should this be removed altogether?

Reply:
I left it in commented out because it's a line that is there from a vanilla rails install.  My theory is that this makes it easier to see what's a change we've made vs what's a new bit of config when we do future rails upgrades.

Reply:
👍